### PR TITLE
Add gnmi-cli.py support

### PR DESCRIPTION
**Pull request by mhwang408**
@Yi-Tseng

### CODE REVIEW

File: tools/gnmi/gnmi-cli.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def str2bool(v):
                     type=str, action="store", required=False)
 parser.add_argument('--interval', help='[Sample subcribe only] Sample subscribe poll interval in ms',
```
Comment (Yi-Tseng) [inline tools/gnmi/gnmi-cli.py]: same as above

File: tools/gnmi/README.md [hunk 2]
```diff
@@ -28,17 +29,21 @@ optional arguments:
                         [SetRequest only] Set string value
   --float-val FLOAT_VAL
                         [SetRequest only] Set float value
+  --bytes-val BYTES_VAL
+                        [SetRequest only] Set bytes value
+  --interval INTERVAL   [Sample subcribe only] Sample subscribe poll interval in ms
```
Comment (Yi-Tseng) [inline tools/gnmi/README.md]: oops, should be `subscribe`

Comment (mhwang408) [reply tools/gnmi/README.md]: Done.

File: tools/gnmi/gnmi-cli.py (hunk 1 above)
Comment (mhwang408) [reply tools/gnmi/gnmi-cli.py]: Done.

**Pull request merged**